app/src/main/java/battleship/Player.java

```diff
@@ -152,6 +161,18 @@ public List<Ship> getShips() {
     }
 
     /**
+<<<<<<< HEAD
```

Comment:
I think we could remove the git-related comments here (<<<<<< HEAD, >>>>>>> mvp)